data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1477 @@
+			`"You got it, <last>," he replies without hesitation. "After all, you're bending over backwards to make sure one of my best friends is safe and sound. I'll let my other buddies know to keep an eye out for you - the more feelers we can put out, the better."`
+			`The pair of you soon exit your ship and head back to the auxiliary captains. "I won't let you down, Lieutenant," Henry says as he gets back in line. "Thanks for giving me this chance!"`
+			label end
+			`It takes a couple hours before all the mechanics have loaded onto the auxiliaries. With your errand complete, you board the <ship> and get ready to head back to Geminus.
+
+	on complete
+		payment 2500000
+		dialog
+			`As you break through the clouds over Geminus, you are treated to the awe-inspiring sight of the new shipyard unfold before you. "Welcome to the new Geminus, <last>," Danforth congratulates you. "This represents the first step of many on the road to victory. To celebrate, take these 2.5 million credits - I'm sure there's something in the shipyard you might want."`
+
+event "geminus rebuilt"
+	planet Geminus
+		description `Geminus, although not the most hospitable of worlds, is a young enough planet that large stores of iron, titanium, and other metals are easily accessible near the surface. The constant volcanic activity leaves the planet perpetually shrouded in a twilight haze of ash and fog.`
+		description `	The shattered remnants of the old Navy Yard are still left abandoned and slowly rusting away in the planet's caustic atmosphere - but now that the Republic has completed a new shipyard to replace the one destroyed in the bombing, Geminus has once again become a central hub of technology and industry.`
+		spaceport `Like everything else on Geminus, the spaceport is not terribly pretty, but is functional and efficient. Each of the wide hallways is marked out like a road, with space in the center for electric carts carrying cargo, while pedestrians are relegated to the edges. Video cameras and sensors have been placed at every intersection, and access to the shipbuilding facilities is tightly controlled.`
+		spaceport `	The hub of the spaceport is a food court with a ring of shops in the center and seating on the outside, where you can look out of plate glass windows at the lava fields and caldera outside. By night, you can see the volcanic glow all around the horizon.`
+		shipyard "Basic Ships"
+		shipyard "Navy Basics"
+		shipyard "Navy Advanced"
+		shipyard "Betelgeuse Basics"
+		shipyard "Lionheart Basics"
+		outfitter "Common Outfits"
+		outfitter "Ammo North"
+		outfitter "Lovelace Advanced"
+		outfitter "Syndicate Basics"
+
+
+mission "Tier Boost"
+	landing
+	source
+		government "Republic"
+	to offer
+		has "Geminus 3C: done"
+		has event "geminus rebuilt"
+	on offer
+		log "Was promoted because of my assistance with rebuilding the shipyards on Geminus."
+		"reputation: Republic" += 10
+		"reputation: Free Worlds" = 500
```

Comment:
I assume this is just because there's that rep-reset at the end of FW, right?
Otherwise, there's no reason to be friendly with them, seeing as you're at war with them.